doofinder-for-woocommerce/includes/class-rest-api-handler.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Doofinder\WP;
+
+use \WP_HTTP_Response;
+use \WP_REST_Server;
+use \WP_REST_Request;
+
+class REST_API_Handler
+{
+    /**
+     * Add the actions and filters needed to modify REST requests
+     *
+     * @return void
+     */
+    public static function initialize(): void
+    {
+        add_action('rest_post_dispatch', array(__CLASS__, 'add_taxonomy_image_link'), 99, 3);
```

Comment:
Just a question: Should this be triggered if the plugin is not active?

Reply:
I left it outside the WooCommerce check as it is a WordPress endpoint, but thinking it twice, it should be called only from WooCommerce clients, so it makes sense changing it to trigger only when Woocommerce is enabled.
Thank you for your appreciation.